feature/funding/googleplay/src/main/res/values/strings.xml

```diff
@@ -1,13 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="funding_googleplay_contribution_title">Settings</string>
```

Comment:
Why "Settings" and not e.g. "Support Thunderbird"?

Reply:
It's in the design, but I'll change it in a follow up pull-request.